src/gui/programs/waddy.rs

```diff
@@ -548,7 +548,14 @@ impl WaddyGui {
                             if is_search_enabled {
                                 self.instances[instance_index].texture_tiles[texture_tile]
                                     .name
-                                    .contains(self.instances[instance_index].search.text.as_str())
+                                    .to_lowercase()
+                                    .contains(
+                                        self.instances[instance_index]
+                                            .search
+                                            .text
+                                            .to_lowercase()
```

Comment:
Pull the whole `self.instances[instance_index].search.text.lowercase()` outside of the loop so that it is not calculated many times over.